### ISSUE TITLE
Preload link doesn't work

### ISSUE THREAD

**Issue by ViktoryiaYatskova** [mdn/sprints#2758]
In Chrome (v79.0.3945.130) and Safari (v 13.0.4 / 14608.4.9.1.4) the warning "<link rel=preload> must have a valid `as` value" occurs in the console and preloading doesn't work.

**Comment by jadjoubran** [comment]
Hey @ViktoryiaYatskova 
I encountered this bug 2 days ago so I've got some details on that.
It turns out that preloading video does not work (anymore?) on Chrome, Firefox & Safari and indeed shows a warning on Chrome & Safari.
There's an open bug for Chrome here: [1076609](https://bugs.chromium.org/p/chromium/issues/detail?id=1076609&q=preload%20video&can=2)
I couldn't find bug reports for other browsers

The preload [spec](https://www.w3.org/TR/preload/) shows that as="video" is part of the spec
so in conclusion, does it make sense that this is not an issue with MDN docs?

**Comment by chrisdavidmills** [comment]
@jadjoubran I think adding a warning would make sense.

**Comment by jadjoubran** [comment]
Added a note under [What types of content can be preloaded](https://wiki.developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content#What_types_of_content_can_be_preloaded)

**Comment by chrisdavidmills** [comment]
Looks good, thanks @jadjoubran ! Let's close this now.
